README.md

```diff
@@ -1,82 +1,177 @@
 
-Azure IoT Device Workbench aims to support multiple popular IoT development boards and kits. It currently supports following IoT hardware:
+## Device platforms and languages supported
+
+Azure IoT Device Workbench aims to support multiple device platforms and its devices. Currently the following device platforms and languages are supported :
+
+### Embedded Linux <sup>preview</sup>
+
+Languages supported: `C/C++`
+
+Devices supported: [Cortex-A series](https://developer.arm.com/ip-products/processors/cortex-a) devices (e.g. Raspberry Pi, NXP i.MX6) that are running Embedded Linux such as Debian, Ubuntu or Yocto Linux.
+
+### Arduino
+
+Languages supported: `Arduino C/C++`
+
+Devices supported:
 
 - [MXChip IoT DevKit](https://aka.ms/iot-devkit)
-- [teXXmo IoT button](https://aka.ms/button)
-- [Raspberry Pi](https://www.raspberrypi.org/)
 - [ESP32](https://www.espressif.com/en/products/hardware/esp-wroom-32/overview)
+- [teXXmo IoT button](https://aka.ms/button)
 
-Please find the [Setup Guide](#setup-guide) section below to configure your hardware.
-
-We will support more devices in Azure IoT Device Workbench. Please help to take the [survey](https://www.surveymonkey.com/r/C7NY7KJ) to let us know your opinion of the device.
+Please help to take the [survey](https://www.surveymonkey.com/r/C7NY7KJ) to let us know extra device platforms and languages you want to have for Device Workbench.
 
 ## Installation
-Open VS Code and press `F1` or `Ctrl + Shift + P` to open command palette, select **Install Extension** and type `iot-workbench`.
 
-Or launch VS Code Quick Open (`Ctrl + P`), paste the following command, and press enter.
-```bash
-ext install vscode-iot-workbench
-```
+### Prerequisites
 
-After installation, please reload the window and make sure all the dependency extensions are correctly installed.
+For developing on Embedded Linux device, you will need Docker and Visual Studio Code version with [Visual Studio Code Remote Development](https://aka.ms/vscode-remote) to compile the device code in the [containerized device toolchain](https://aka.ms/iot-device-cube).
 
-## Supported Operating Systems
-Currently this extension supports the following operatings systems:
+- [Visual Studio Code](https://code.visualstudio.com/), with version >= 1.35.1.
+- [Docker Desktop / CE](https://www.docker.com/get-started).
 
-- Windows 7 and later (32-bit and 64-bit)
-- macOS 10.10 and later
+### Install extensions
 
-## Setup Guide
+1. Launch VS Code, in the Extension tab, find and install **"Azure IoT Device Workbench"**.
 
-### MXChip IoT DevKit
+    ![Extension install Device Workbench](./docs/images/device-workbench.png)
 
-Please follow the [guide](docs/iot-devkit.md) to setup IoT DevKit.
+2. Reload the window and make sure all the dependency extensions are correctly installed:
 
-Here are a set of tutorials that will help you get started:
+   - Azure Account
+   - Azure IoT Hub Toolkit
+   - C/C++
+   - IoT Device Cube
+   - Remote Development
```

Comment:
Remote Development Extension will be lazy installed instead of as a dependent extension which will be installed at the very beginning. Only when user chooses to create a container project will we ask user to install Remote Development Extension from marketplace.